debian/changelog

```diff
@@ -1,3 +1,96 @@
+flatpak (0.11.3-3endless0) unstable; urgency=medium
+  * New upstream releases
+  * Rebase the eight Endless patches
```

Comment:
The debian packaging does not know about these patches; I'd rather drop this comment.

Reply:
Sure